plugins/ingest-attachment/build.gradle

```diff
@@ -23,7 +23,7 @@ esplugin {
 }
 
 versions << [
-  'tika': '1.14',
+  'tika': '1.15',
```

Comment:
There's more that will be needed than this, for example, updating the SHAs. Did you run `gradle check`?